rust/toolchain.bzl

```diff
@@ -60,6 +60,7 @@ def _rust_stdlib_filegroup_impl(ctx):
             std_files = std_files,
             alloc_files = alloc_files,
             self_contained_files = self_contained_files,
+            all_targets = ctx.attr.srcs,
```

Comment:
Nit: I'd go with `all_files` to be consistent with other attributes here (and also to be consistent with `cc_toolchain`).

Reply:
As discussed offline, reverted back to using the original `srcs` attribute for this since it's a list of targets, which makes it a bit different from the other attributes of this provider.